cmd/clusterctl/examples/openstack/provider-component/user-data/ubuntu/templates/master-user-data.sh

```diff
@@ -94,9 +94,12 @@ function getversion() {
 }
 KUBELET=$(getversion kubelet ${KUBELET_VERSION}-)
 KUBEADM=$(getversion kubeadm ${KUBELET_VERSION}-)
+KUBECTL=$(getversion kubectl ${KUBELET_VERSION}-)
 apt-get install -y \
     kubelet=${KUBELET} \
-    kubeadm=${KUBEADM}
+    kubeadm=${KUBEADM} \
```

Comment:
@hchenxa Seems the `kubeadm init` can help to install `kubectl` as well, can you help check if the `kubeadm init` will overwrite the `kubectl` installed by `apt-get install`?

Reply:
CentOS has same logic as https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/master/cmd/clusterctl/examples/openstack/provider-component/user-data/centos/templates/master-user-data.sh#L62 , I think it is OK to have this PR.